components/src/molecules/product-list-item/ProductListItem.js

```diff
@@ -0,0 +1,97 @@
+import AImage from '../../atoms/image/Image.vue'
+import AIcon from '../../atoms/icon/Icon.vue'
+import APrice from '../../atoms/price/Price.vue'
+import AList from '../../atoms/list/List.vue'
+import AListItem from '../../atoms/list-item/ListItem.vue'
+import AButton from '../../atoms/button/Button.vue'
+
+// @vue/component
+export default {
+  components: {
+    AImage,
+    AIcon,
+    APrice,
+    AList,
+    AListItem,
+    AButton
+  },
+  model: {
+    prop: 'qty',
+    event: 'change'
+  },
+  props: {
+    /**
+     * Product name
+     */
+    name: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product url (string value or URL)
+     */
+    url: {
+      type: [URL, String],
+      required: true
+    },
+    /**
+     * Product image url
+     */
+    imageUrl: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product standard price
+     */
+    price: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product special price
+     */
+    specialPrice: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product old price
+     */
+    oldPrice: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product options/attributes array (size, color etc)
+     */
+    options: {
+      type: Array,
+      default: null
+    }
+  },
+  computed: {
+    /**
+     * Dynamically generated item id based on name and timestamp
+     */
+    id () {
+      return `${new Date().getTime()}_${String(this.name).toLowerCase().replace(/\s/g, '-')}`
```

Comment:
Do we need it? Imo every product should has id in data :)

Reply:
You're right, moved to props.